src/views/Delegate.vue

```diff
@@ -273,7 +292,28 @@ onMounted(async () => {
         </router-link>
         <h1 v-if="loaded" v-text="$t('delegate.header')" />
       </div>
-      <template v-if="loaded">
+      <PageLoading v-if="!loaded" />
+      <Block v-if="!networkSupportsDelegate">
+        <Icon name="warning" class="mr-1" />
+        {{
+          $t('delegate.delegateNotSupported', {
+            network:
+              networks?.[networkKey]?.shortName ?? $t('theCurrentNetwork')
+          })
+        }}
+        <div>
```

Comment:
Can we remove this div so that the Learn More is on the same line?

Reply:
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/51686767/152634767-75bbcc1b-45b0-467c-b583-58362f5cef67.png">